app/forms/mailing_address_form.rb

```diff
@@ -12,11 +12,14 @@ def save
   end
 
   def self.existing_attributes(intake)
-    if intake.street_address.present?
-      attributes = intake.attributes
-    else
-      attributes = intake.primary_user.attributes
-    end
+    attributes = if intake.street_address.present?
+                   intake.attributes
+                 elsif intake.primary_user.present?
+                   intake.primary_user.attributes
+                 else
+                   {}
+                 end
```

Comment:
This is great. We could also set state to state_of_residence at this point too, just to make things easier for the user.